src/gpkg/gpkg.ts

```diff
@@ -17,21 +17,22 @@ export class Gpkg {
   private readonly config: IConfig;
   private readonly extent: BBox2d;
   private readonly maxZoomLevel: number;
+  private readonly packageName: string;
   private gpkgConfig?: IGpkgConfig;
 
-  public constructor(path: string, extent: BBox2d, zoomLevel: number) {
+  public constructor(path: string, extent: BBox2d, zoomLevel: number, packageName: string) {
```

Comment:
as packageName recieved via swagger need to validate swagger for invalid chars in file name

Reply:
trigger should do this, and validate invalid chars before insert it into the parameters in db